salt/metalk8s/bootstrap/calico.sls

```diff
@@ -0,0 +1,2 @@
+include:
+  - metalk8s.calico
```

Comment:
I don't really understand why we have all these 'aliases' for existing states. What's the use for these?

Reply:
It was mapped against the functions we had in bootstrap.sh, and maybe it will be easier to pick just what we want to add a new nodes to the control plane.

Reply:
OK. At some point I guess we'll want to simplify/unify the states we have though.